.circleci/config.yml

```diff
@@ -119,3 +136,18 @@ jobs:
       - *postgres_service
       - *mysql_service
     <<: *unit_tests
+  publish_npm:
+    docker:
+      - image: node:8
+        user: node
+    steps:
+      - checkout
+      - run:
+          name: Set NPM authentication.
+          command: echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
```

Comment:
For this to work, you have to set the `NPM_TOKEN` env var in the circleci setting. The token that we're using is in valentine.

Reply:
Done. Thanks!